apps/meteor/app/e2e/client/rocketchat.e2e.ts

```diff
@@ -538,6 +538,31 @@ class E2E extends Emitter {
 		return decryptedMessageWithQuote;
 	}
 
+	async decryptPinnedMessage(message: IMessage) {
```

Comment:
Can we write unit test for this?

Reply:
Even better, added e2e tests for entire flow.